cmd/kots/cli/install.go

```diff
@@ -59,6 +61,15 @@ func InstallCmd() *cobra.Command {
 
 			log := logger.NewCLILogger()
 
+			if !v.GetBool("skip-rbac-check") && v.GetBool("ensure-rbac") {
+				err := CheckRBAC()
+				if err != nil {
+					log.Error(errors.New("Current user has insufficient privileges to install Admin Console. For more information, please visit https://kots.io/vendor/packaging/rbac"))
```

Comment:
Can you add a screenshot of what this looks like in the terminal?

Reply:
![Screen Shot 2021-05-20 at 12 12 42 PM](https://user-images.githubusercontent.com/27239392/119036168-4d05e700-b965-11eb-9956-a1cd15e82bc8.png)

Reply:
![Screen Shot 2021-05-20 at 12 51 16 PM](https://user-images.githubusercontent.com/27239392/119040315-1f6f6c80-b96a-11eb-8c25-04ce07f715f1.png)

Reply:
![Screen Shot 2021-05-20 at 12 53 16 PM](https://user-images.githubusercontent.com/27239392/119040591-737a5100-b96a-11eb-9fd0-50d11c1c8689.png)

Reply:
![Screen Shot 2021-05-20 at 12 55 29 PM](https://user-images.githubusercontent.com/27239392/119040867-c8b66280-b96a-11eb-9293-a42ed96401a7.png)